ui/src/components/MotorInput/BaseMotorInput.jsx

```diff
@@ -63,9 +61,8 @@ function BaseMotorInput(props) {
 
   return (
     <form className="d-flex" noValidate onSubmit={handleSubmit}>
-      <Form.Control
```

Comment:
Bootstrap's styles kind of get in the way of our customisation needs, so I prefer to remove them by switching to a bare `input`.